Lib/idlelib/debugger.py

```diff
@@ -1,14 +1,23 @@
 import bdb
 import os
+import linecache
 
 from tkinter import *
-from tkinter.ttk import Frame, Scrollbar
+from tkinter import ttk
+from tkinter import PhotoImage
```

Comment:
Unused import...

Reply:
Our current IDLE practice is to explicitly import all widgets.  The replacement of the current ttk import is a regression.  Please revert.  The new tkinter import is redundant with the * import, but I would like the * to be replaced by an explicit list.

Reply:
done

Reply:
The separate `PhotoImage` import is unneeded (`from tkinter import BooleanVar, Menu, PhotoImage`)